packages/pillarbox-playlist/scss/pillarbox-playlist.scss

```diff
@@ -0,0 +1,47 @@
+.pbw-playlist-button {
+  cursor: pointer;
+}
+
+.vjs-modal-dialog.pbw-playlist-dialog {
+  right: 0;
+  left: auto;
+  width: 30em;
```

Comment:
We could use `width:100%` and `max-width; 30em`. This would make it possible to adapt to the size of the player without necessarily having to play with breakpoints, which could be left to the discretion of integrators.